aws/route53/nameservers

```diff
@@ -26,7 +26,8 @@ if test -z "$HUB_DOMAIN_NAME"; then
   exit 1
```

Comment:
Maybe `color e` instead of `echo`

Reply:
color e will redirect to stderr

Reply:
I meant an error message in line 25. Replace `echo` with `color e`

Reply:
yes, done!